styles.css

```diff
@@ -0,0 +1,190 @@
+/* style for body is temporary */
+body {
+  max-width: max-content;
+  margin: auto;
+  border: 20px solid black;
+  border-top-left-radius: 20px;
+  border-top-right-radius: 20px;
+}
+
+.container {
+  width: 375px;
+  height: 6715px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.top-bar {
+  width: 375px;
+  height: 92px;
+  display: flex;
+  flex-direction: column;
+}
+
+.status {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 44px;
+}
+
+#time-container {
+  width: 54px;
+  height: 21px;
+  margin-top: 13px;
+  margin-left: 21px;
+  margin-bottom: 12px;
+}
+
+#status-container {
+  width: 66px;
+  height: 11px;
+  margin-top: 17px;
+  margin-right: 15px;
+}
+
+.header {
+  display: flex;
+  width: 375px;
+  height: 48px;
+  justify-content: space-between;
+}
+
+.text-welcome {
+  width: 114px;
+  height: 48px;
+  margin-left: 12px;
+  align-content: center;
+}
+
+.welcome {
+  width: 90px;
+  height: 28px;
+  font-size: 20px;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  margin-left: 12px;
+  margin-right: 12px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+a:link {
+  text-decoration: none;
+  color: black;
+}
+
+.humberger-icon-container {
+  width: 40px;
+  height: 40px;
+  margin-right: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 726px;
+  align-items: center;
+  background-image: url("images/header-illsutration-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-size: 375px 726px;
+}
+
+.primary-text {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 193px;
+  margin-top: 64px;
+  text-align: center;
+  font-size: 40px;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
```

Comment:
[ ] kindly use the same `font-family` as the one used in the design 🤓.

![font](https://user-images.githubusercontent.com/62156607/192011444-7b0c899f-9458-401e-90f9-2b057720bff1.PNG)

Reply:
ok.